requirements/prod.in

```diff
@@ -33,5 +33,5 @@ psycopg2-binary==2.9.2
 rq-scheduler==0.11.0
 sentry-sdk==1.5.0
 service_identity==21.1.0
-https://github.com/SFDO-Tooling/sfdo-template-helpers/archive/v0.18.0.tar.gz
+https://github.com/SFDO-Tooling/sfdo-template-helpers/archive/v0.19.0.tar.gz
```

Comment:
@jerivas I'm not sure, but I'm guessing the failing pytest relates to changes in https://github.com/SFDO-Tooling/sfdo-template-helpers/pull/41?

Reply:
Fixed in 918ce83b57fc0f4b7f1c6d1aec042cee5c94cae4